data/uri-options/tls-options.yml

```diff
@@ -1,7 +1,7 @@
 tests:
     -
         description: "Valid required tls options are parsed correctly"
-        uri: "mongodb://example.com/?tls=true&tlsCAFile=ca.pem&tlsCertificateKeyFile=cert.pem&tlsCertificateKeyFilePassword=hunter2"
+        uri: "mongodb://example.com/?tls=true&tlsCAFile=ca.pem&tlsCertificateKeyFile=cert.pem"
```

Comment:
Driver spec tests come in YAML and JSON because different languages have different parsers available to them in the standard library or external modules. For Go, we only run the JSON files, but the YAML is often helpful because it can contain comments or explanations for a test, which JSON does not allow, so we keep both sets in the driver.